.flowconfig

```diff
@@ -13,6 +13,9 @@ static/src/javascripts/__flow__/types
 ####### ATOM LIBRARY #######
 node_modules/@guardian/atom-renderer/dist/__flow__/types
 
+####### DOTCOM RENDERING #######
+node_modules/@guardian/dotcom-rendering/flow-typed
+
```

Comment:
@GHaberis - does this need to be in here?

Reply:
I think because we’re using the Flow-annotated source code of dotcom-rendering, we need to tell Flow where to find types for the dotcom-rendering package. When we properly publish guui and pasteup, we can remove this line.

I might be wrong however, so feel free to try removing  it anyway and see if it’s necessary